Foundation/NSGeometry.swift

```diff
@@ -459,8 +476,7 @@ public func NSIntegralRectWithOptions(_ aRect: NSRect, _ opts: NSAlignmentOption
     if aRect.size.width.native < 0 {
         width = 0
     }
-    
-
+#if !os(Android) // FIXME!!!
```

Comment:
I think we need to understand what's going on here; I don't really want to see a // FIXME in code. JIRA number, or a fatalError of some kind (like our NSUnimplemented).